templates/docs/document.html

```diff
@@ -1,118 +1,140 @@
-          {% if not nav_group.hidden %}
-            {% if nav_group.navlink_text %}
-              {% if nav_group.navlink_href %}
-                <h3 class="p-side-navigation__heading--linked">
-                  <a class="p-side-navigation__link" href="{{ nav_group.navlink_href }}" {% if nav_group.is_active %}aria-current="page"{% endif %}>
-                    {{ nav_group.navlink_text }}
-                  </a>
-                </h3>
-              {% else %}
-                <h3 class="p-side-navigation__heading">{{ nav_group.navlink_text }}</h3>
+            {% if not nav_group.hidden %}
+              {% if nav_group.navlink_text %}
+                {% if nav_group.navlink_href %}
+                  <h3 class="p-side-navigation__heading--linked">
+                    <a class="p-side-navigation__link"
+                       href="{{ nav_group.navlink_href }}"
+                       {% if nav_group.is_active %}aria-current="page"{% endif %}>{{ nav_group.navlink_text }}</a>
+                  </h3>
+                {% else %}
+                  <h3 class="p-side-navigation__heading">{{ nav_group.navlink_text }}</h3>
+                {% endif %}
               {% endif %}
-            {% endif %}
-            {#
+              {#
               Use `create_navigation(nav_group.children)` for a default, fully expanded navigation.
               Use `create_navigation(nav_group.children, expandable=True)` for the nested nav levels to expand only when parent page is active.
             #}
-            {{ create_navigation(nav_group.children, expandable=True) }}
-          {% endif %}
+              {{ create_navigation(nav_group.children, expandable=True) }}
+            {% endif %}
           {% endfor %}
         </div>
       </nav>
-    </aside>
-    <main class="col-9" id="main-content">
+    </div>
+  </div>
+  <div class="l-docs__title">
+    <div class="u-fixed-width">
       <h1>{{ document.title }}</h1>
+    </div>
+  </div>
+  {% if document.headings_map is defined and document.headings_map|length > 0 %}
+    <div class="l-docs__meta">
+      <div class="l-docs__sticky-container">
+        <aside class="p-table-of-contents">
+          <div class="p-table-of-contents__section">
+            <h4 class="p-table-of-contents__header">On this page</h4>
+            <nav class="p-table-of-contents__nav" aria-label="Table of contents">
+              <ul class="p-table-of-contents__list">
+                {% for heading in document.headings_map %}
+                  <li class="p-table-of-contents__item">
+                    <a class="p-table-of-contents__link" href="#{{ heading.heading_slug }}">{{ heading.heading_text }}</a>
+                  </li>
+                {% endfor %}
+              </ul>
+            </nav>
+          </div>
+        </aside>
+      </div>
+    </div>
+  {% endif %}
+  <!-- {{ document }} -->
+  <div class="l-docs__main">
+    <main id="main-content u-fixed-width">
```

Comment:
```suggestion
  <div class="l-docs__main u-fixed-width">
    <main id="main-content">
```
The left and and right margins were not being properly applied. This should fix it.